task/08-objects-tasks.js

```diff
@@ -23,10 +23,11 @@
  *    console.log(r.getArea());   // => 200
  */
 function Rectangle(width, height) {
-    throw new Error('Not implemented');
+    this.width = width;
+    this.height = height;
+    Rectangle.prototype.getArea = () => width * height;
```

Comment:
Please, implement this line outside Rectangle function, because everytime function Rectange be invoked - every time prototype will be changed